tests/packages.py

```diff
@@ -552,5 +552,23 @@ def runTest(self):
             self.assertEqual(len(w), 0)
             self.assertEqual(str(self.handler.packages), "\n%packages --retries=10\nbash\n\n%end\n")
 
+class Packages_Warn_CamelCase_TestCase(ParserTest):
+    def __init__(self, *args, **kwargs):
+        ParserTest.__init__(self, *args, **kwargs)
+        self.version = F32
+        self.ks = """
```

Comment:
May as well make this all one line with embedded \n like you did in the assert check.